pkg/oci/remote/remote.go

```diff
@@ -156,26 +156,26 @@ func attachment(digestable digestable, attName string, o *options) (oci.File, er
 		return nil, fmt.Errorf("expected exactly one layer in attachment, got %d", len(ls))
 	}
 
-	return &attache{
+	return &attached{
 		SignedImage: img,
 		layer:       ls[0],
 	}, nil
 }
 
-type attache struct {
+type attached struct {
```

Comment:
I'd be tempted to use `attachment` but I didn't want to have to figure out if golang would scream at me.

I'm not entirely certain what this object is.